subprojects/language-java/src/main/java/org/gradle/external/javadoc/internal/JavadocOptionFileWriterContext.java

```diff
@@ -95,6 +95,28 @@ public JavadocOptionFileWriterContext writeMultilineValuesOption(String option,
         return this;
     }
 
+    public JavadocOptionFileWriterContext writeMultiValueOption(String option, Collection<String> values) throws IOException {
+        writeOptionHeader(option);
```

Comment:
Fixed a minor bug in `MultilineMultiValueJavadocOptionFileOption` described [here](https://github.com/gradle/gradle/issues/6991#issuecomment-427278068)